### ISSUE TITLE
Python 3.12: DeprecationWarning: datetime.datetime.utcfromtimestamp() is deprecated

### ISSUE THREAD

**Issue by mrakitin** [caproto/caproto#838]
In [our testing](https://github.com/NSLS-II-SRX/srx-caproto-iocs/actions/runs/7918485797/job/21617052186?pr=2) of a caproto IOC with Python 3.12 we are observing the following issue:
```py
ImportError while loading conftest '/home/runner/work/srx-caproto-iocs/srx-caproto-iocs/tests/conftest.py'.
tests/conftest.py:13: in <module>
    from srx_caproto_iocs.base import OphydDeviceWithCaprotoIOC
/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/srx_caproto_iocs/base.py:11: in <module>
    from caproto import ChannelType
/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/caproto/__init__.py:2: in <module>
    from ._utils import *
/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/caproto/_utils.py:29: in <module>
    from ._dbr import SubscriptionType
/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/caproto/_dbr.py:17: in <module>
    from ._constants import (EPICS2UNIX_EPOCH, EPICS_EPOCH, MAX_ENUM_STATES,
/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/caproto/_constants.py:11: in <module>
    EPICS_EPOCH = datetime.datetime.utcfromtimestamp(EPICS2UNIX_EPOCH)
E   DeprecationWarning: datetime.datetime.utcfromtimestamp() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.fromtimestamp(timestamp, datetime.UTC).
Error: Process completed with exit code 4.
```
Is it a known problem?

**Comment by tacaswell** [comment]
https://github.com/matplotlib/matplotlib/pull/25918/files is how we fixed it on Matplotlib.

**Comment by klauer** [comment]
Closed by #843
